src/jumper/__init__.py

```diff
@@ -7,10 +7,10 @@
 
 
 def jump():
```

Comment:
? pressing 8 is somehow jumping? -> just kidding, ill change this to be able to change the key value dynamically.
Maybe you should keep the random interval, something like
time.sleep(randint(30,40))
This would make it a tiny bit less repetitive!

Reply:
Since there is no jump everything is allowed xD changed for random interval, but is repetitive necessarily a bad thing?